project_tests.sh

```diff
@@ -3,6 +3,7 @@ set -e
 
 source venv/bin/activate
 
-pylint --reports=n manage.py profiles/ test/*.py
-flake8 manage.py profiles/ test/
+pip install proofreader==0.0.2
```

Comment:
This means that doing a `pip freeze` will see it added to `requirements.txt`?

Reply:
If you use `project_tests.sh` locally I think so, usually it is called by Jenkins only.

Reply:
Yes, in local development you would then see it added if you use `project_tests.sh`. 

Though `pip freeze` is clunky as it expands sub-dependencies of actual dependencies, really we should be managing the requirement files a bit more tightly (I am working on this) for example currently there are x60 dependencies listed in the `requirements.txt`.

Reply:
Just checking (something to watch out for).

Definitely not a fan of the current `requirements.txt`.